cloud-optimized-geotiffs/writing-cogs-in-python.ipynb

```diff
@@ -0,0 +1,547 @@
+{
```

Comment:
<p>Please add back a citation to the source of the information.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/106/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>